### ISSUE TITLE
Modification of add_terminator in cfg.py

### ISSUE THREAD

**Issue by tissue3** [sampsyo/bril#28]
Consider a typescript program looks like this:
```
let value = 8;
let result = 1;
for (let i = value; i > 0; i = i - 1) {
  if (i>5){
    result = result * i;
  }
}
```
The if statement of this program will be translated to bril like this
```
  br v10 then.7 else.7;
then.7:
  v11: int = id result;
  v12: int = id i;
  v13: int = mul v11 v12;
  result: int = id v13;
  jmp endif.7;
else.7: 
endif.7:
  v14: int = id i;
...
```
The translation is correct. But block `else.7` is empty.
However, because in our own project we would like to take use of `add_terminators` in cfg.py, we find `block[-1]['op'] not in TERMINATORS` does not work since block length of `else.7` is empty and there is no `block[-1]`.
It would be better to modify the function to
```
def add_terminators(blocks):
    for i, block in enumerate(blocks.values()):
        if len(block) == 0:
            dest = list(blocks.keys())[i + 1]
            block.append({'op': 'jmp', 'args': [dest]})
        elif block[-1]['op'] not in TERMINATORS:
        ...
```

**Comment by sampsyo** [comment]
Aha, yes, good point! This is a good idea! Please open a PR with this change.

One Python nit: `if not block:` is a shorter & clearer way to say `if len(block) == 0:`.

**Comment by sampsyo** [comment]
Actually, wait a sec—changing the condition to this would be easier:

```
if not block or block[-1]['op'] not in TERMINATORS:
```

That is, just add `not block or` to the beginning of that condition. Then you don't have to duplicate any code, and it also works when the empty block is the last one in the function.

**Comment by sampsyo** [comment]
Fixed in #29!
